started
ERROR: Windmill requires that you provide a starting URL...
[workspace] $ windmill safari test=~/Desktop/crap/test_google.py exit
windmill.browser.safari: INFO     ['/Applications/Safari.app/Contents/MacOS/Safari', '/var/folders/YO/YOAvf2hSGt0+VnBIAKuImE+++TM/-Tmp-/tmp38sqnz.html']
2009-04-29 16:58:36.680 Safari[70731:10b] *** CFMessagePort: bootstrap_register(): failed 1100 (0x44c) 'Permission denied', port = 0xe003, name = 'com.apple.Safari.ServiceProvider'
See /usr/include/servers/bootstrap_defs.h for the error codes.
test_results: ERROR    Test Failure in test {'version': '0.1', 'suite_name': u'test_google', 'result': False, 'starttime': u'2009-3-29T16:58:45.616Z', 'output': None, 'debug': u'Looking up name q, failed.', u'params': {u'text': u'testing', u'name': u'q', 'uuid': 'ac484c86-3519-11de-a8f2-002332da2796'}, 'endtime': u'2009-3-29T16:58:45.619Z', u'method': u'type'}
windmill.authoring: ERROR    Functest test failed: test_recordingSuite0
Leopard libedit detected.
Server running...
Traceback (most recent call last):[39;49;00m
  File [39;49;00m[36;01m"/Users/adam/Desktop/crap/test_google.py"[39;49;00m, line [39;49;00m[36;01m7[39;49;00m, in [39;49;00mtest_recordingSuite0[39;49;00m
    [39;49;00mclient[39;49;00m.[39;49;00mtype[39;49;00m([39;49;00mtext[39;49;00m=[39;49;00m[33mu'[39;49;00m[33mtesting[39;49;00m[33m'[39;49;00m,[39;49;00m [39;49;00mname[39;49;00m=[39;49;00m[33mu'[39;49;00m[33mq[39;49;00m[33m'[39;49;00m)[39;49;00m
  File [39;49;00m[36;01m"/Users/adam/Documents/projects/windmill/branches/1.1.x/windmill/authoring/__init__.py"[39;49;00m, line [39;49;00m[36;01m162[39;49;00m, in [39;49;00m__call__[39;49;00m
    [39;49;00m[36;01mreturn[39;49;00m [39;49;00m[36;01mself[39;49;00m.[39;49;00mexec_method[39;49;00m([39;49;00m[36;01mself[39;49;00m.[39;49;00maction_name[39;49;00m,[39;49;00m [39;49;00m*[39;49;00m*[39;49;00mkwargs[39;49;00m)[39;49;00m
  File [39;49;00m[36;01m"/Users/adam/Documents/projects/windmill/branches/1.1.x/windmill/authoring/__init__.py"[39;49;00m, line [39;49;00m[36;01m215[39;49;00m, in [39;49;00m_exec_test[39;49;00m
    [39;49;00m[36;01mraise[39;49;00m [39;49;00mWindmillTestClientException[39;49;00m([39;49;00mresult[39;49;00m[[39;49;00m[33m'[39;49;00m[33mresult[39;49;00m[33m'[39;49;00m][39;49;00m)[39;49;00m
[04m[32;01mWindmillTestClientException: {u'suite_name': u'test_google', u'result': False, u'starttime': u'2009-3-29T16:58:45.616Z', u'params': {u'text': u'testing', u'name': u'q'}, u'debug': u'Looking up name q, failed.', u'output': None, u'endtime': u'2009-3-29T16:58:45.619Z', u'method'[39;49;00m: [39;49;00mu'type'}[39;49;00m

finished: FAILURE
